### PR TITLE
docs: improve network usage instructions

### PR DESCRIPTION
**Pull request by mattstam**
Clarify the instructions to using `cast` and getting whitelisted.

### CODE REVIEW

File: book/prover-network/usage.md [hunk 1]
```diff
@@ -17,8 +17,8 @@ SP1_PROVER=network SP1_PRIVATE_KEY=... RUST_LOG=info cargo run --release
 
 - `SP1_PROVER` should be set to `network` when using the prover network.
 
-- `SP1_PRIVATE_KEY` should be set to your [private key](#key-setup). You will need
-  to be using a [permissioned](#get-access) key to use the network.
+- `SP1_PRIVATE_KEY` should be set to your [private key](./setup.md#key-setup). You will need
```
Comment (ctian1) [inline book/prover-network/usage.md:20]: These links should be `./setup` I think: https://docs.succinct.xyz/prover-network/setup

You can run mdbook locally to test, check readme

File: book/prover-network/setup.md [hunk 2]
```diff
@@ -6,30 +6,43 @@ Succinct [has been building](https://blog.succinct.xyz/succinct-network/) the Su
 
 ## Get access
 
-Currently the network is permissioned, so you need to gain access through Succinct. After you have completed the key setup below, you can submit your address in this [form](https://docs.google.com/forms/d/e/1FAIpQLSd-X9uH7G0bvXH_kjptnQtNil8L4dumrVPpFE4t8Ci1XT1GaQ/viewform) and we'll contact you shortly.
+**[FILL OUT THIS FORM](https://forms.gle/rTUvhstS8PFfv9B3A)** to gain access to the Succinct
```
Comment (ctian1) [inline book/prover-network/setup.md:9]: IMO this was better as it was before

File: book/prover-network/setup.md [hunk 3]
```diff
@@ -6,30 +6,43 @@ Succinct [has been building](https://blog.succinct.xyz/succinct-network/) the Su
...
-[Install](https://book.getfoundry.sh/getting-started/installation#using-foundryup):
+First [install Foundry](https://book.getfoundry.sh/getting-started/installation#using-foundryup):
 
 ```sh
 curl -L https://foundry.paradigm.xyz | bash
 ```
 
-Generate a new keypair:
+after running this command, you will be prompted to source your shell profile and run `foundryup`.
+After that completes, you will have access to the `cast`.
```
Comment (ctian1) [inline book/prover-network/setup.md:26]:
```suggestion
Upon running this command, you will be prompted to source your shell profile and run `foundryup`. Afterwards you should have access to the `cast` command.
```

File: book/prover-network/setup.md (hunk 2 above)
Comment (mattstam) [reply book/prover-network/setup.md:9]: @ratankaliani said that it was too hard to find the form

File: book/prover-network/usage.md (hunk 1 above)
Comment (mattstam) [reply book/prover-network/usage.md:20]: I tested them, they're correct.

**Review by ctian1 — APPROVED** [review]

**Pull request merged**